package.json

```diff
@@ -0,0 +1,54 @@
+{
+  "name": "shopping-list-app",
+  "version": "1.0.0",
+  "description": "This is a shopping list app that allows for the creation of various lists with their own items. It uses Html, Css, Jquery, Javascript, and LocalStorage.",
+  "main": "js/index.js",
+  "dependencies": {
+    "jquery": "^3.0.0",
```

Comment:
Personal choice here, but I always remove the carrots, and set the version. Especially with dev dependencies. I know these set of versions work together, and I don't want to come back in a few.months to a broken config because 1 updated